app/code/Transoft/Blog/Model/DataProvider.php

```diff
@@ -0,0 +1,95 @@
+<?php
+declare(strict_types=1);
+
+namespace Transoft\Blog\Model;
+
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Filesystem\Io\File;
+use Magento\Ui\DataProvider\AbstractDataProvider;
+use Transoft\Blog\Model\ResourceModel\Blog\CollectionFactory;
+
+/**
+ * Blog model ui data provider
+ */
+class DataProvider extends AbstractDataProvider
+{
+    /**
+     * @var array
+     */
+    private $_loadedData;
```

Comment:
Please do not use underscore for field and argument names. Fix all

Reply:
fixed